src/uint/modular/mod.rs

```diff
@@ -1,17 +1,62 @@
-use crate::UInt;
```

Comment:
This PR looks good aside from the use of the 2015 edition module style (i.e. `mod.rs`)

We've adopted the 2018 edition module style, where this file would be modular.rs.

Unfortunately I don't think they ever added a lint for mixed module styles.

Reply:
Is this what you mean?:
https://rust-lang.github.io/rust-clippy/master/index.html#mod_module_files

Reply:
Oh ho, went looking for that the other day and didn't find it... weird